api/r/property_label.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: Property Label
+nav_order: 3
+parent: R
+grand_parent: API
+---
+
+# Show Property Labels of Node(s)
+
+## `get_property_labels(dcids, out=True)`
+
+Returns the labels of properties defined for the given node DCIDs
+
+**Arguments**
+
+*    `dcids` (`vector` of strings) - DCIDs identifying nodes to query.
+
+*   `out` (`logical`, optional) - Whether or not the property points away from the given list of nodes.
+
+**Returns**
+
+A *named* `vector` mapping DCIDs to `vector`s of property labels. If `out` is `TRUE`, then property labels correspond to edges directed away from given nodes. Otherwise, they correspond to edges directed towards the given nodes.
+
+**Raises**
+
+*   `Response error` - If the payload returned by the Data Commons REST API is malformed or the API key is not set.
+
+Be sure to initialize the library, and specify the API key. Check the [R library setup guide](/api/r/) for more details.
+
+## Examples
+
+**Examples**
+
+To get all outgoing property labels for [California](https://browser.datacommons.org/kg?dcid=geoId/06>) and
+[Colorado](https://browser.datacommons.org/kg?dcid=geoId/08), we can write the following:
+
+```r
+> library(datacommons)
+> set_api_key(YOUR_API_KEY_HERE)
+> get_property_labels(c('geoId/06', 'geoId/08'))
+$`geoId/06`
+[1] "containedInPlace" "geoId"            "kmlCoordinates"  
+[4] "name"             "provenance"       "typeOf"          
+
+$`geoId/08`
+[1] "containedInPlace" "geoId"            "kmlCoordinates"  
+[4] "name"             "provenance"       "typeOf"
+
```

Comment:
nit: remove empty lines here (and below)

Reply:
R returns new lines for output but I agree, not needed :)